src/js/components/tasks/steps.tsx

```diff
@@ -0,0 +1,77 @@
+import classNames from 'classnames';
+import i18n from 'i18next';
+import _ from 'lodash';
+import React, { useEffect, useState } from 'react';
+
+import { Task } from '@/store/tasks/reducer';
+import { TASK_STATUSES } from '@/utils/constants';
+
+interface TaskStatusPathProps {
+  task: Task;
+}
+
+const TaskStatusSteps = ({ task }: TaskStatusPathProps) => {
+  const [completedSteps, setcompletedSteps] = useState<number[]>([]);
+  const [nextSteps, setNextSteps] = useState<number[]>([]);
+
+  useEffect(() => {
+    const inProgressDone = task.has_unmerged_commits ? [0, 1] : [0, 1, 2, 3];
+    const inProgressNext = task.has_unmerged_commits ? [3] : [4];
+    let done: number[], next: number[];
+    switch (task.status) {
+      case TASK_STATUSES.PLANNED:
+        done = [];
+        next = [0, 1];
+        break;
+      case TASK_STATUSES.IN_PROGRESS:
+        done = inProgressDone;
+        next = inProgressNext;
+        break;
+      case TASK_STATUSES.COMPLETED:
+        done = [0, 1, 2, 3, 4, 5]; // or maybe all the indexes of steps ?
+        next = [];
+        break;
+    }
+    setcompletedSteps(done);
+    setNextSteps(next);
+  }, [task]);
+  const steps = [
+    `${i18n.t('Assign developer')}`,
+    `${i18n.t('Assign reviewer')}`,
+    `${i18n.t('Create a Scratch Org for development')}`,
+    `${i18n.t('Capture task changes')}`,
+    `${i18n.t('Submit task changes for review')}`,
+    `${i18n.t('Create a QA Org for development')}`,
+  ];
+  // @todo render different steps for login as dev or reviewer
+
```

Comment:
There are different steps displayed based on who the logged in user is (either as dev or qa).  Since this is in progress, Could I perhaps look to that branch to see how this can mocked for now?

Reply:
@duggiemitchell I'm reviewing this code, but you're correct -- I think it'll be more efficient to set this PR on hold until `task-assign` is complete, then merge that branch in here before finishing it.